src/main/kotlin/xyz/atrius/waystones/data/config/Localization.kt

```diff
@@ -11,15 +11,24 @@ import java.util.*
 class Localization(val plugin: KotlinPlugin, localeTag: String) {
     private var configFile = File(plugin.dataFolder, "locale-$localeTag.yml")
     private val config = YamlConfiguration()
-    private var locale: Locale = Locale.getDefault()
+    var locale: Locale = Locale.getDefault()
 
     init {
         reload(localeTag)
     }
 
     fun localize(key: String, vararg args: Any?): String {
-        return MessageFormat(config.getString(key)!!, locale)
-                .format(*args.map { it ?: "null" }.toTypedArray()).translateColors()
+        val template = config.getString(key)
+                ?: throw IllegalArgumentException("$key does not exist in localization file!")
+        return format(template, *args)
+    }
+
+    fun format(string: String, vararg args: Any?): String {
+        return MessageFormat(string, locale).format(args).translateColors()
+    }
+
+    operator fun get(key: String, vararg args: Any?): String {
+        return localize(key, *args)
```

Comment:
You mentioned before that we should avoid passing `null` into the formatting methods, it might be a good idea to enforce that at compile-time as well by switching out `Any?` with just `Any`

Reply:
There are a lot of varaible could be null (while actually never is) in current code, changing them seems not in the scope of this PR.

Reply:
That's fair